yt_dlp/extractor/downdogapp.py

```diff
@@ -0,0 +1,191 @@
+import time
+import urllib
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    urlencode_postdata,
+)
+
+
+class DownDogAppIE(InfoExtractor):
+    SUBDOMAINS = ['www', 'meditation', 'pilates', 'hiit', 'barre', 'prenatal']
+    _VALID_URL = rf'https?://({"|".join(SUBDOMAINS)})\.downdogapp\.com/play#(?P<id>[^\?]*)'
+    _LOGIN_URL_PATH = '/json/login'
+    _MANIFEST_URL_PATH = '/manifest'
```

Comment:
since `_LOGIN_URL_PATH` and `_MANIFEST_URL_PATH` are only used once in their own specific methods, they should be inlined